### PR TITLE
Update statisticsRoutes.js

### PR DESCRIPTION
**Pull request by twegestal**
Added input validation and try/catch blocks in statisticRoutes.js

### CODE REVIEW

File: apps/server/src/routes/statisticsRoutes.js [hunk 1]
```diff
@@ -7,39 +7,59 @@ export const getStatisticRoutes = () => {
   const router = Router();
```
Comment (LektorVinberg) [inline apps/server/src/routes/statisticsRoutes.js:8]: Please add a doc-style comment up top

**Review by LektorVinberg — CHANGES_REQUESTED** [review]
See comment on line 9

File: apps/server/src/routes/statisticsRoutes.js (hunk 1 above)
Comment (twegestal) [reply apps/server/src/routes/statisticsRoutes.js:12]: Fixed!

**Review by LektorVinberg — APPROVED** [review]
Looks gid

**Pull request merged**